test/integration/client-side-operations-timeout/client_side_operations_timeout.prose.test.ts

```diff
@@ -1,8 +1,48 @@
 /* Specification prose tests */
 
+import { type ChildProcess, spawn } from 'node:child_process';
+
+import { expect } from 'chai';
+import * as os from 'os';
+import * as path from 'path';
+import * as semver from 'semver';
+import * as sinon from 'sinon';
+import { Readable } from 'stream';
+import { pipeline } from 'stream/promises';
+
+import { type CommandStartedEvent } from '../../../mongodb';
+import {
+  Binary,
+  ClientEncryption,
+  type CommandSucceededEvent,
+  GridFSBucket,
+  MongoBulkWriteError,
+  MongoClient,
+  MongoOperationTimeoutError,
+  MongoServerSelectionError,
+  now,
+  ObjectId,
+  promiseWithResolvers,
+  squashError
+} from '../../mongodb';
+import { type FailPoint, makeMultiBatchWrite, measureDuration } from '../../tools/utils';
+import { filterForCommands } from '../shared';
+
 // TODO(NODE-5824): Implement CSOT prose tests
```

Comment:
RM TODO. 4 is intentionally omitted.

Reply:
Still need to remove TODO

Reply:
Addressed in https://github.com/mongodb/node-mongodb-native/pull/4315/files